ietf/static/css/liaisons.css

```diff
@@ -1,67 +1,4 @@
-.baseform { 
-    font-size: 12px;
-}
-
-.baseform .fieldset {
-    margin: 1em 0px;
-    border: none;
-    border: 1px solid #8899dd;
-    background-color: #edf5ff;
-}
-
-.baseform .fieldset h2 {
-    background-color: #2647a0;
-    color: white;
-    font-size: 14px;
-    padding: 5px 10px;
-    margin: 0px;
-}
-
-.baseform .field {
-    padding: 0.5em 10px;
-}
-
-.baseform .field label {
-    display: block;
-    width: 150px;
-    float: left;
-    clear: left;
-}
-
-.baseform .field .endfield {
-    clear: left;
-}
-
-.baseform .fieldWidget {
-    margin-left: 150px;
-}
-
-.baseform #baseform-fieldname-purpose_text,
-.baseform #baseform-fieldname-deadline_date {
-    display: none;
-}
-
-.baseform select,
-.baseform textarea,
-.baseform input {
-    border: 1px solid #cccccc;
-}
-
-.baseform input {
-    font-size: 12px;
-}
-
-#id_title,
-.baseformedit #id_from_field,
-.baseform #id_organization,
-.baseform #id_to_poc,
-.baseform #id_response_contacts,
-.baseform #id_technical_contact,
-.baseform #id_cc1,
-.attach_titleField input,
-.baseform textarea {
-    width: 80%;
-}
+/* TODO: this should use bs5 styling and ideally be integrated into the code */
```

Comment:
What tooling are you using that shows the removed styling was not used?

Reply:
I grepped for "baseform" in the repo :-)